src/App.tsx

```diff
@@ -118,8 +118,10 @@ class App extends React.Component<IAppProps, any> {
     }
     this.setState(newState);
 
+    // note: not an actual hash, this is just for checking if the query has been done before. Consider renaming it
```

Comment:
The part of an URL after a # is sometimes called hash

Reply:
That's right, still `fragment` would be more [official](https://tools.ietf.org/html/rfc3986#section-3.5).